File: 1171.c
#include <stdio.h>

int main(void) {
    int q, i, n;
    int lista[2000];

    for (i = 0; i < 2000; ++i)
        lista[i] = 0;

    scanf("%d", &q);
    for (i = 0; i < q; ++i) {
        scanf("%d", &n);
        ++lista[n-1];
    }

    for (i = 0; i < 2000; ++i)
        if (lista[i] != 0)
        printf("%d aparece %d vez(es)\n", i+1, lista[i]);
    return 0;
}

File: 2-ad-hoc/1171.py
n = int(input())
v = []
for i in range(n):
    v.append(int(input()))
v.sort()
for i, j in enumerate(v):
    if i == 0:
        print('{} aparece {} vez(es)'.format(j, v.count(j)))
    elif j != u: print('{} aparece {} vez(es)'.format(j, v.count(j)))
    u = j
